docker/project.clj

```diff
@@ -33,16 +33,23 @@
                  [org.clojure/java.data "1.0.86"]
                  [potemkin "0.4.5"]
                  [reply "0.4.4" :exclusions [javax.servlet/servlet-api]]
-                 [zero.one/fxl "0.0.5"]]
+                 [zero.one/fxl "0.0.5"]
+                 [org.apache.arrow/arrow-memory-netty "2.0.0"]
```

Comment:
Perhaps this should be moved to provided dependencies? (all the Arrow deps I mean)

Reply:
done